### PR TITLE
feat: [redis bus]implement test for system events

### LINKED ISSUE #695: Implement automation test for device system event
# 🚀 Feature Request

### Relevant Package [**REQUIRED**]

 This feature request is for Core Metadata

### Description [**REQUIRED**]
 
This is a placeholder for implementing test for https://docs.edgexfoundry.org/2.3/design/adr/0024-system-events/
There will be more details once this ADR is implemented.

### PR DESCRIPTION
**Pull request by jumpingliu**
close #695 

Signed-off-by: melody <melody@iotechsys.com>

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [ ] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [x] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

File: TAF/testScenarios/integrationTest/UC_system_events/metadata_system_events_redis.robot [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+*** Keywords ***
+Update Multiple Devices
+    Update Device Command-Device-0 With operatingState=DOWN
+    Update Device Command-Device-1 With adminState=LOCKED
+    Update Device Command-Device-2 With label=test-system-event
+
+Single System Event Should Match With Source ${source}, Type ${type} and Action ${action}
+    ${received_content}  grep file  ${WORK_DIR}/TAF/testArtifacts/logs/${subscriber_file}  Payload
+    Should Not Be Empty  ${received_content}
```
Comment (cherrycl) [inline TAF/testScenarios/integrationTest/UC_system_events/metadata_system_events_redis.robot:71]:
```suggestion
    Wait Until Keyword Succeeds  5x  1s  File Should Not Be Empty  ${WORK_DIR}/TAF/testArtifacts/logs/${subscriber_file}
    ${received_content}  grep file  ${WORK_DIR}/TAF/testArtifacts/logs/${subscriber_file}  Payload
```
Also use on multiple keyword.

Comment (jumpingliu) [reply TAF/testScenarios/integrationTest/UC_system_events/metadata_system_events_redis.robot]: Thanks for the review comment. Updated.
I will open aother PR on mqtt bus tests.

Comment (jumpingliu) [reply TAF/testScenarios/integrationTest/UC_system_events/metadata_system_events_redis.robot]: updated

**Review by cherrycl — APPROVED** [review]
LGTM

**Pull request merged**